commit a63b96fbddbf97feaa068a9efed3b5623a1a1e78
Author: Vincent Chen <vincentc@andestech.com>
Date:   Wed Jun 26 17:30:11 2019 +0800

    Linux: Add nds32 specific syscalls to syscall-names.list
    
    The nds32 creates two specific syscalls, udftrap and fp_udfiex_crtl, in
    kernel v5.0 and v5.2, respectively. Add these two syscalls to
    syscall-names.list.

diff --git a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
index ae8adabb70..95aa3ec7a5 100644
--- a/sysdeps/unix/sysv/linux/syscall-names.list
+++ b/sysdeps/unix/sysv/linux/syscall-names.list
@@ -121,6 +121,7 @@ finit_module
 flistxattr
 flock
 fork
+fp_udfiex_crtl
 free_hugepages
 fremovexattr
 fsetxattr
@@ -603,6 +604,7 @@ tkill
 truncate
 truncate64
 tuxcall
+udftrap
 ugetrlimit
 ulimit
 umask
